Outcomes/utils.py

```diff
@@ -0,0 +1,90 @@
+import datetime
```

Comment:
True. I was planning on merging with and importing SEIR.utils once I'm done developing (because it takes time to install the package every time, but perhaps python dev have a way of using a module without installing that I'm not aware of). But now that you've brought this up 2. and 4. are cool.

Good thing with 2. is that I would be faster and less disk I/O as it would be possible to run without saving SEIR sims. Then it's a big change...

Reply:
I imported from SEIR in the meantime